scripts/setup

```diff
@@ -47,5 +45,10 @@ cds access allow superuser-access role admin
 # Create a default files location
 cds files location videos /tmp/files
 
-# Load demo records
+# Load demo data
+cds fixtures sequence-generator
+cds fixtures categories
+cds fixtures pages
+cds fixtures keywords
+cds fixtures licenses
 # cds fixtures records
```

Comment:
thanks for fixing, I am always using my local scripts instead of fixing this one.